### ISSUE TITLE
Thread view shows messages outside of follow graph no matter their type

### ISSUE THREAD

**Issue by KyleMaas** [arj03/ssb-browser-demo#251]
Thread view does not filter messages based on type, which means messages may be shown as outside of follow graph even if we don't know their type (whether they're posts or not) or are private.

**Comment by arj03** [comment]
Right, that should be relatively easy to fix though

**Comment by KyleMaas** [comment]
Yep.  Just didn't want to forget about it.

**Comment by KyleMaas** [comment]
Actually, this might not be so simple.  Messages outside of the follow graph are known about only by their IDs by being mentioned in other messages.  Which means without fetching them, we can't get their type.  So...I'm not sure this is actually possible to fix.

**Comment by arj03** [comment]
Right, but we can implement some kind of warning system for that, and maybe a small cache so that you don't have to fetch the messages again. This would be very important for blocked users.

**Comment by KyleMaas** [comment]
I think I'm going to be a bit out of my league on that - mind taking this one?

**Comment by arj03** [comment]
Sure, a bit busy now but I'll try keeping this in mind

**Comment by KyleMaas** [comment]
Yeah, no problem.  Just trying to go through old bugs and see which ones are still valid.
